thesis/inc/requirements/performance.tex

```diff
@@ -0,0 +1,59 @@
+\section{Performance}
+As stated earlier, \todo{State this earlier} performance is an important aspect within the games industry,
+as a result of this, certain restrictions and wishes were decided upon by between Suttung and the Authors.
+
+\subsection{Virtual Functions}
+\label{subsec:requirements_performance_virtual_functions}
+\todo{Refer to Game Engine Architecture?}
+Virtual functions is C++'s way of supporting dynamic function binding,allowing for run-time polymorphism.
+With the help of virtual functions, objects derived from a base class can override base class functionality,
+even when derived's type is unknown at compile time.
+The implementation of virtual functions are not mandated by the C++ ISO Standard,
+however, many compilers choose the same tactic; creating a virtual table.\cite{wikipedia_virtual_method_table}
+\todo{Link to standard? (c++14 or c++17)?, Ask Mariusz if draft is ok, as the real thing costs money}
+
+A virtual table can be viewed as a data structure containing function pointers to virtual functions that should be called in a derived class.
```

Comment:
Replace should with will. I'm quite confident that the virtual functions will be called in the derived classes. Could also restructure to just saying "... functions that is called in a derived class."